purebred.cabal

```diff
@@ -20,17 +21,23 @@ description:
   See HACKING for more info.
 
 homepage:            https://github.com/githubuser/purebred#readme
-license:             AGPL-3
+license:             AGPL-3.0-or-later
 license-file:        LICENSE
 author:              Róman Joost & Fraser Tweedale
 maintainer:          frase@frase.id.au
-copyright:           2017  Purebred contributors
+copyright:           2017-2018  Purebred contributors
 category:            Email
 build-type:          Simple
-cabal-version:       >=1.10
-extra-source-files:  README.md
+
+-- these wildcard expressions require cabal >= 2.4
+extra-source-files:
+  README.md
+  test/data/Maildir/**/*.url
+  "test/data/Maildir/**/*.url:2,RS"
+  configs/purebred.hs
+
 tested-with:
-  GHC==8.0.2, GHC==8.2.2
+  GHC==8.0.2, GHC==8.2.2, GHC==8.4.4, GHC==8.6.3
```

Comment:
Was wondering if we shouldn't drop 8.0.2? In a way I'd be happy to stick with whatever is currently in Fedora ...

Reply:
Might as well keep it in the mix until it actually gets in the way of something.